src/govee_ble/parser.py

```diff
@@ -374,6 +363,45 @@ def _process_mfr_data(
             )
             return
 
+        if msg_length == 20 and (
+            "H5198" in local_name
+            or mgr_id == 0x3022
+            or "00009851-0000-1000-8000-00805f9b34fb" in service_uuids
+        ):
+            self.set_device_type("H5198")
+            self.set_device_name(f"H5198 {short_address(address)}")
+            sensor_id = data[6]
+            (
+                temp_probe_first,
+                temp_alarm_first,
+                _,
+                temp_probe_second,
+                temp_alarm_second,
+            ) = PACKED_hhhhh.unpack(data[8:18])
+            if sensor_id in [0x01, 0x41, 0x81, 0xC1]:
+                ids = [1, 2]
+            elif sensor_id in [0x02, 0x42, 0x82, 0xC2]:
+                ids = [3, 4]
```

Comment:
```python
H5198_PROBE_MAPPING_1_2 = [1, 2]
H5198_PROBE_MAPPING_3_4 = [3, 4]

H5198_PROBE_MAPPING = {
	{sensor_id: H5198_PROBE_MAPPING_1_2 for sensor_id in (0x01, 0x41, 0x81, 0xC1)} |
	{sensor_id: H5198_PROBE_MAPPING_3_4 for sensor_id in (0x02, 0x42, 0x82, 0xC2)} |
}
```

```suggestion
            if not (ids := H5198_PROBE_MAPPING.get(sensor_id)):
```

Reply:
I changed it to a general 4-probe solution, as we can do the same for the H5184.